model/src/cfd.rs

```diff
@@ -940,7 +952,9 @@ impl Cfd {
             "Failed to propose collaborative settlement"
         );
 
-        let payout_curve = calculate_payouts_long_taker(
+        let payout_curve = calculate_payouts(
+            self.position,
+            self.role,
             self.initial_price,
             self.quantity,
             self.long_leverage,
```

Comment:
was unable to comment on the line below...

We still always hard-code `Leverage::new(SHORT_LEVERAGE)` - this has to be change to `self.short_leverage` if I understand this correctly. 

Note that this is in multiple places!
I'd recommend removing the `SHORT_LEVERAGE` constant. The maker always being locked in to `Leverage::ONE` should already be correctly handled when constructing the `Cfd`.

Reply:
Nice catch!

Reply:
Makes sense! I'll add a commit on top because this is not strictly related to assigning the payouts to the correct parties.

Reply:
Would be great if you could remove the constant `SHORT_LEVERAGE` so we ensure that we don't use it anymore.

Reply:
> Would be great if you could remove the constant `SHORT_LEVERAGE` so we ensure that we don't use it anymore.

Pretty sure I did!

Reply:
Ah, yeah, sorry. Somhow did not see it in the commit 😅